packages/gestalt/src/ExpandableModuleBase.jsdom.test.js

```diff
@@ -0,0 +1,74 @@
+// @flow strict
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import ExpandableModuleBase from './ExpandableModuleBase.js';
+
+describe('ExpandableModuleBase', () => {
+  let props = {
+    title: 'test title',
+    summary: ['summary1', 'summary2', 'summary3'],
+    isCollapsed: true,
+    onModuleClicked: jest.fn(),
+    type: 'info',
+  };
+
+  it('should render the collapsed state correctly', () => {
+    render(<ExpandableModuleBase {...props}>Children</ExpandableModuleBase>);
+    expect(screen.getByText(/test title/i)).toBeInTheDocument();
+    expect(screen.queryByText(/summary1/i)).toBeInTheDocument();
+    expect(screen.queryByText(/Children/i)).toBeNull();
+
+    const expandButton = screen.getByRole('button', {
+      name: /arrow icon to expand or collapse the module/i,
+    });
+    fireEvent.click(expandButton);
+    expect(props.onModuleClicked).toHaveBeenCalledWith(false);
+  });
+
+  it('should render icon correctly', () => {
+    props = {
```

Comment:
@galaxy0101 instead of mutating props, can we make it more explicit?

```
const propsWithIcon = {
   ...baseProps,
   icon: 'lock',
   iconAccessibilityLabel: 'test label'
}
```

Reply:
fixed